### PR TITLE
CAV-392 - Gateway - Passcode persisting failure & passcode verificati…

### PR DESCRIPTION
**Pull request by johno1985**
…on check failure returns incorrect status code & message

 - added serverselectiontimeout param to mongo connection string

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -89,7 +89,7 @@ auditing {
 }
 
 mongodb {
-  uri = "mongodb://localhost:27017/cip-phone-number-verification"
+  uri = "mongodb://localhost:27017/cip-phone-number-verification?serverselectiontimeoutms=10000"
```
Comment (brdgeth) [inline conf/application.conf:92]: I think we should make the timeout configurable, i.e. as a separate config variable and not part of the url.  Then refer to it here in the url.

Comment (johno1985) [reply conf/application.conf:92]: Agreed, and well spotted, I forgot to raise PR for the app-config-{env} repos. Will raise some more shortly

Comment (brdgeth) [reply conf/application.conf]: thanks for making them configurable now.  
One question, in other environments does mongodb always run on port 27017?  I think there are a range of ports you can choose from, 27018 is another one for example.  Therefore do you think you should make the port number configurable as well?

Comment (brdgeth) [reply conf/application.conf:92]: ok fair enough.   No worries, just leave as is, the way you had it originally.

**Comment by brdgeth** [comment]
I will approve it once you fix the build issue, you have a failing test.

**Comment by johno1985** [comment]
rebuild

**Pull request merged**